src/App.js

```diff
@@ -35,10 +35,11 @@ const App = (props) => {
         insights.chrome?.globalFilterScope?.('insights');
         if (insights.chrome?.globalFilterScope) {
             insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
-                const [workloads, SID, selectedTags] = insights.chrome?.mapGlobalFilter?.(data, false, true);
```

Comment:
Can you revert this change? As the new API of `mapGlobalFilter` allows passing 3rd parameter to preformat the data for you? https://github.com/RedHatInsights/insights-chrome/blob/f0eb20df8671392fc0d3ef9aad6eb40b64659fb9/README.md#prefered-way-of-consuming-data-in-structured-format

Reply:
this change breaks the dashboard workload selector (neither inventory or advisor cards work with workloads) furthermore, at least in prod-beta, SIDs dont exist yet n this change gobbles up tags, shows em as SIDS

Reply:
also, regarding the link sent, is this work on prod beta? not seeing     // workloads = { SAP: { isSelected: true } } as the default value returned when using the mapGlobalFilter

Reply:
I forgot the API is still not merged to each environment... Yeah we can roll out with the older version of mapping global filter and I'll open new PR to enable the newer way once the API lands.

Reply:
The formatted API is just on ci/qa-beta and is blocked by API unfortunately.